picasso/src/main/java/com/squareup/picasso3/BitmapHunter.java

```diff
@@ -376,6 +379,7 @@ static BitmapHunter forRequest(Picasso picasso, Dispatcher dispatcher,
     return new BitmapHunter(picasso, dispatcher, cache, stats, action, ERRORING_HANDLER);
   }
 
+  @SuppressWarnings("NullAway")
```

Comment:
Not nullable?

Reply:
This method...yea, the posted messages to the Handler crash on invalid input anyway.  This needs cleanup...